types/insane/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "rules": {
```

Comment:
All of these rules are in place for good reasons, and there aren't even violations of most of these you've disabled here. Please remove as many of these disables as you can.

Reply:
Done